chess-AI/experimental-work/output_representation.py

```diff
@@ -66,12 +65,10 @@ def __init__(self):
 
     def convert_policy_indices_to_uci_move(self, indices, board_t):
         '''Construct move in UCI format from indices corresponding to neural network output cell.
-
```

Comment:
There should be a blank line here per [function docstring format](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)